client/coral-admin/src/components/BanUserDialog.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import {Dialog} from 'coral-ui';
+import Button from 'coral-ui/components/Button';
+
+import styles from './BanUserDialog.css';
+
+import I18n from 'coral-framework/modules/i18n/i18n';
+import translations from '../translations';
+const lang = new I18n(translations);
+
+const BanUserDialog = ({open, handleClose, onClickBanUser, user}) => {
+  const userName = (typeof user === 'undefined') ? '' : user['userName'];
+  const userId = (typeof user === 'undefined') ? '' : user['userId'];
+  const commentId = (typeof user === 'undefined') ? '' : user['commentId'];
```

Comment:
these three lines can become:
`const {userName = '', userId = '', commentId = ''} = user;`
since `user` is always defined